public/js/clients/index.js

```diff
@@ -13,7 +13,23 @@ function getClient() {
   return firefox.clientCommands;
 }
 
-function startDebugging(targetEnv, tabId, actions) {
+function startDebugging(connTarget, actions) {
+  if (connTarget.type === "node") {
+    return startDebuggingNode(connTarget.param, actions);
+  }
+
+  const target = connTarget.type === "chrome" ? chrome : firefox;
+  return startDebuggingTab(target, connTarget.param, actions);
+}
+
+function startDebuggingNode(url, actions) {
```

Comment:
I added `startDebuggingNode` to be parallel with `startDebuggingTab`. Also because `clientType` needs to be set for the actions to use the right client.